app/src/main/res/layout/activity_maps.xml

```diff
@@ -29,7 +29,7 @@
                     android:id="@+id/map"
                     android:name="com.google.android.gms.maps.SupportMapFragment"
                     android:layout_width="match_parent"
-                    android:layout_height="match_parent" />
+                    android:layout_height="match_parent"/>
```

Comment:
![screen shot 2017-04-27 at 12 57 48](https://cloud.githubusercontent.com/assets/7342988/25492368/391f4320-2b49-11e7-9769-c31133809751.png)
Acredito eu que seja aquele spaces in empty tag.